bigbluebutton-html5/imports/api/cursor/server/handlers/cursorUpdate.js

```diff
@@ -1,24 +1,32 @@
 import { check } from 'meteor/check';
 import CursorStreamer from '/imports/api/cursor/server/streamer';
+import Logger from '/imports/startup/server/logger';
+
+
+const { streamerLog } = Meteor.settings.private.serverLog;
 
 const CURSOR_PROCCESS_INTERVAL = 30;
 
-let cursorQueue = [];
-let cursorRecieverIsRunning = false;
+let cursorQueue = {};
+let cursorReceiverIsRunning = false;
 
 const proccess = () => {
   if (!Object.keys(cursorQueue).length) {
-    cursorRecieverIsRunning = false;
+    cursorReceiverIsRunning = false;
     return;
   }
-  cursorRecieverIsRunning = true;
+  cursorReceiverIsRunning = true;
 
-  Object.keys(cursorQueue).forEach((meetingId) => {
-    CursorStreamer(meetingId).emit('message', { meetingId, cursors: cursorQueue[meetingId] });
-  });
-  cursorQueue = {};
+  try {
+    Object.keys(cursorQueue).forEach((meetingId) => {
+      CursorStreamer(meetingId).emit('message', { meetingId, cursors: cursorQueue[meetingId] });
+    });
+    cursorQueue = {};
 
-  Meteor.setTimeout(proccess, CURSOR_PROCCESS_INTERVAL);
+    Meteor.setTimeout(proccess, CURSOR_PROCCESS_INTERVAL);
+  } catch (error) {
+    cursorReceiverIsRunning = false;
```

Comment:
Would it not be a good idea to log the error that is caught. Or even just log that the catch was needed.